.travis.yml

```diff
@@ -27,16 +27,12 @@ before_install:
       npm install -g codecov
     fi
 install:
-  - pip install --upgrade -e ".[test]"
-  - |
-    if [[ $GROUP == js ]]; then
-      cd nbdime-web
-      npm install
```

Comment:
Note: The removal of the `npm install` line should also reveal if that step was not completed during `pip install`.